.circleci/config.yml

```diff
@@ -339,6 +339,34 @@ jobs:
             RUST_BACKTRACE=1 $CARGO $CARGOFLAGS build -j 16 -p transaction-builder -p move-vm-types --target powerpc-unknown-linux-gnu
       - save_sccache
       - build_teardown
+  build-individual-crates:
+    executor: build-executor
+    parallelism: 7
+    description: Build all top level individual crates
+    steps:
+      - build_setup
+      - restore_cargo_package_cache
+      - install_sccache
+      - restore_sccache
+      - run:
+          name: Determine crate targets for this container.
+          command: |
+            for x in $(find . -name Cargo.toml); do dirname $x >> /tmp/crates; done
```

Comment:
we really do need an xbuild that can build all top level members for us, as a one shot or separately.   Something like cargo xbuild --no-feature-unification  p1 p2 p3. Where each one would be built independently.   I'll add this to the xbuild card.

Reply:
I'd wager a target determinator - through the dependency graph (such as guppy), it can determine which target(s) need to re-build.  

Maybe that's the next big thing to tackle? cc @metajack